src/main/java/aquarium/shop/basket/Basket.java

```diff
@@ -0,0 +1,95 @@
+package aquarium.shop.basket;
+
+import aquarium.shop.product.Product;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+
+public class Basket {
+    private static Basket INSTANCE;
+
+    private Basket() {
+    }
+
+    private HashMap<Product, Integer> content = new HashMap<>();
+    private double wholePrice;
```

Comment:
You can think of changing **double** type to **BigDecimal** here, it's better for prices.